bin/mock-outcomes

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python
```

Comment:
Could you please add some information in the header of this file as to what the purpose of it is? _If_ it applies, also list if there is any set up to do before executing this script.

Reply:
This is a script I got from @maxiuyuan , it was kinda useful but I'm not sure if we should include it in this PR.